services/src/main/java/mops/SingleSubmitService.java

```diff
@@ -0,0 +1,39 @@
+package mops;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import mops.rollen.Student;
+@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+public class SingleSubmitService {
```

Comment:
Not a good Name for a Service. The functionality is not very different from just normally submitting. This naming is confusing